report_data/templates/email/approved_rd.html

```diff
@@ -12,7 +12,7 @@ <h2 class="text-title">{% if lang == "es" %} Notificación de Registro Aprobado.
             en el SINAMECC, con el ID {{rd_code}} ha sido Aprobada. ¡Felicitaciones!
             </p>
             <p>
-            Puede consultar el estado de su solicitud en el siguiente $enlace$. Agradecemos el registro de <br/>
+            Puede consultar el estado de su solicitud en el siguiente <A HREF={{frontend_url}}/report/view/{{id_report}}> enlace</A>. Agradecemos el registro de <br/>
```

Comment:
Change tags to lowercase: `<a href='something'></a>` and  
`report_id` instead of `id_report`